Remove commas for explanations about how to set API keys in concerned cogs

PredaaA opened the pull request:
### Type

- [x] Bugfix
- [ ] Enhancement
- [ ] New feature

### Description of the changes
This PR removes commas in explanations about how to set API keys in streams, image and audio cogs, since it has been changed in #2692.

PredaaA committed:
Update streams.py

PredaaA committed:
Update audio.py

PredaaA committed:
Create 2905.enhance.rst

PredaaA committed:
Create 2905.enhance.rst

Flame442 commented:
The old comma separation does still work, however it may be phased out in the future.

For consistency's sake, imgurcreds in the image cog should also be updated.

PredaaA committed:
Update image.py

PredaaA committed:
Create 2905.enhance.rst

PredaaA commented:
@Flame442 Done, I updated imgurcreds and giphycreds. Also edited my PR comment to include image cog.

Flame442 committed:
Changes from #2729 and #2733

-Replaced in line code blocks with italics
-Removed []() link formatting (for when embeds are disabled)
-Added a period to the doc string
-Added periods at the end of each line
-"Run the command" instead of "Do" for clarity
-"your_api_key_here" instead of "your_api_key" for consistency

PredaaA committed:
Merge pull request #2 from Flame442/patch-1

Changes from #2729 and #2733

PredaaA committed:
Create 2905.enhance.2.rst

Tobotimus committed:
Merge branch 'V3/develop' into patch-3
